partiql-lang/src/test/kotlin/org/partiql/lang/eval/CastTestBase.kt

```diff
@@ -131,6 +131,7 @@ abstract class CastTestBase : EvaluatorTestBase() {
         }
         override fun toString(): String = "$expression -> ${expected ?: expectedErrorCode}"
 
+        // TODO: delete these tests ahead of `v1` release
```

Comment:
(self-review): Within the legacy code of `partiql-lang`, I didn't remove anything just yet. Currently just marks parts of the code in `partiql-lang` that we will remove in the `v1` release.

Reply:
from offline discussion, we'll leave existing tests as-is. They will be deleted in the `v1` release.